src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageStrings.vb

```diff
@@ -63,6 +63,18 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
         Public ReadOnly Property Option_Suppress_hints_when_parameter_names_differ_only_by_suffix As String =
             ServicesVSResources.Suppress_hints_when_parameter_names_differ_only_by_suffix
 
+        Public ReadOnly Property Option_Inline_Diagnostics_experimental As String =
+            ServicesVSResources.Inline_Diagnostics_experimental
```

Comment:
what is thsi string used for?

Reply:
They were not used, leftover from past iterations. removed.